### PR TITLE
update native image ga tag and run lint on all packages when native config file is changed

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
1) Update native image GA to include also the `more_itertools` module.
2) removed the `RegexExtractAll` from ignore list cause it now passes unit-test with the new GA image.
3) Run `lint -a` (run lint on all packages) in case the `docker_native_image_config.json` has been changed.
4) Removed the `Intezer v2` from ignore list.
5) Removed the `PdfLocker` from ignore list.
6) add unit-tests to CheckPointDome9 cause there isn't enough test coverage 

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5259, https://jira-hq.paloaltonetworks.local/browse/CIAC-5398, https://jira-hq.paloaltonetworks.local/browse/CIAC-5241

### CODE REVIEW

File: Tests/docker_native_image_config.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
             "chromium",
             "tesseract"
          ],
-         "docker_ref": "demisto/py3-native:8.1.0.40949"
+         "docker_ref": "demisto/py3-native:8.1.0.45852"
```
Comment (JudahSchwartz) [inline Tests/docker_native_image_config.json:32]: this will be released under 8.1 also? We dont want to use a maintenance image here?

Comment (GuyAfik) [reply Tests/docker_native_image_config.json:32]: why do you mean by "also" - this image replaced completely the old image in 8.1

Comment (JudahSchwartz) [reply Tests/docker_native_image_config.json:32]: theres no rollover?

Comment (GuyAfik) [reply Tests/docker_native_image_config.json:32]: as i said one this image is build, it completely replaces the old one.

**Review by JudahSchwartz — APPROVED** [review]

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**